util/log.py

```diff
@@ -17,15 +17,19 @@
     import user
 
 
-def write(ctx, message):
+def write(ctx, message, write_stdout=False):
     """Write a message to the log, including client name and originating module.
 
-    :param ctx:     Combined type of a callback and rei struct
-    :param message: Message to write to log
+    :param ctx:          Combined type of a callback and rei struct
+    :param message:      Message to write to log
+    :param write_stdout: Whether to write to stdout (used for a few of our scripts)
     """
-    stack = inspect.stack()[1]
-    module = inspect.getmodule(stack[0])
-    _write(ctx, '[{}] {}'.format(module.__name__.replace("rules_uu.", ""), message))
+    if write_stdout:
+        ctx.writeLine("stdout", message)
```

Comment:
We do not want to include the originating module when writing to stdout?

Reply:
The only other time we write to stdout this way as far as I can tell is for the update publications script. And it does not print the originating module either.